pkg/restapi/vc/operation/operations.go

```diff
@@ -861,6 +851,21 @@ func (o *Operation) issueCredentialHandler(rw http.ResponseWriter, req *http.Req
 	o.writeResponse(rw, signedVC)
 }
 
+func (o *Operation) generateKeypairHandler(rw http.ResponseWriter, req *http.Request) {
+	_, signKey, err := o.kms.CreateKeySet()
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusInternalServerError,
+			fmt.Sprintf("failed to create key pair: %s", err.Error()))
+
+		return
+	}
+
+	rw.WriteHeader(http.StatusOK)
+	o.writeResponse(rw, &GenerateKeyPairResponse{
+		PublicKey: signKey,
```

Comment:
Can this be a handle instead?

Reply:
Not sure I understand your comment. Can you please elaborate ?

Reply:
I was referring to a key handle (some opaque reference, eg. uuid). Instead of returning the raw public key contents, can it instead return a reference?

Reply:
The consumer would need the actual public key to create the DID Document.